### PR TITLE
chore(deps): Remove legacy-api guava

### PR DESCRIPTION
**Pull request by cb0s**
### Summary 

chore(deps): Remove guava-google api

### Details 

In a recent discussion with @jvpichowski we realized that we do not seem to use this api, so I decided to remove it. If you, @jantischhoefer also do not use it, please merge this.

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

File: modules/telestion-services/build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ dependencies {
     implementation project(':modules:telestion-api')
 
     implementation group: 'com.fazecast', name: 'jSerialComm', version: '2.7.0'
-    implementation 'com.google.guava:guava:30.1.1-jre'
+//    implementation 'com.google.guava:guava:30.1.1-jre'
```
Comment (jvpichowski) [inline modules/telestion-services/build.gradle:49]: Why comment out? Please remove the line.

**Review by jvpichowski — CHANGES_REQUESTED** [review]

File: modules/telestion-services/build.gradle (hunk 1 above)
Comment (cb0s) [reply modules/telestion-services/build.gradle:49]: Will be done.

**Review by jvpichowski — APPROVED** [review]
Better!

**Review by pklaschka — APPROVED** [review]

**Comment by cb0s** [comment]
Ok, let's wait for @jantischhoefer ... Not that he needs this package ^^

**Comment by jvpichowski** [comment]
> 
> 
> Ok, let's wait for @jantischhoefer ... Not that he needs this package ^^

If he needs it now the test wouldn't succeed..

**Review by jantischhoefer — APPROVED** [review]
As far as i know i have never used this package. Can be merged.

**Pull request merged**